pkg/storage/utils/decomposedfs/revisions.go

```diff
@@ -275,6 +271,15 @@ func (fs *Decomposedfs) RestoreRevision(ctx context.Context, ref *provider.Refer
 	if err != nil {
 		return errtypes.InternalError("failed to copy blob xattrs to old revision to node: " + err.Error())
 	}
+	// always set the node mtime to the current time
+	err = fs.lu.MetadataBackend().SetMultiple(ctx, nodePath,
+		map[string][]byte{
+			prefixes.MTimeAttr: []byte(time.Now().UTC().Format(time.RFC3339Nano)),
+		},
+		false)
+	if err != nil {
+		return errtypes.InternalError("failed to set mtime attribute on node: " + err.Error())
```

Comment:
Use `errors.Wrap`?

Reply:
We just discussed this in person. Wrapping the error would no longer allow for type assertions on the accoring interface
```
type IsInternalError interface {
	IsInternalError()
}
```
which is currently the way to check for certain error types in reva. Thus we'll refrain from wrapping the error for now.